core/beacon/types.go

```diff
@@ -148,6 +148,9 @@ func ExecutableDataToBlock(params ExecutableDataV1) (*types.Block, error) {
 	if len(params.ExtraData) > 32 {
 		return nil, fmt.Errorf("invalid extradata length: %v", len(params.ExtraData))
 	}
+	if len(params.LogsBloom) != 256 {
+		return nil, fmt.Errorf("invalid logsBloom length: %v", len(params.LogsBloom))
+	}
```

Comment:
If we're fixing these to be robust -- one other funky thing that can happen, I think, is that the `params.BaseFeePerGas` can be negative, setting the header basefee to negative. I've no idea what would happen later, but probably nothing good.